test/jdk/java/util/TimeZone/Bug6329116.java

```diff
@@ -245,5 +253,12 @@ static boolean useLocalizedShortDisplayName(TimeZone tz,
 
         return false;
     }
+    static boolean checkEurope_Brussels(String expected, String got) {
+        if ((expected.equals("CET") && got.equals("MET")) || (expected.equals("CEST") && got.equals("MEST")))
+            return true;
+        else
+            return false;
+    }
+
```

Comment:
I don't think this is correct. I think COMPAT's name for "MET" needs to be modified to "CET/CEST", as MET is now a link to Europe/Brussels